src/scripts/main.js

```diff
@@ -1 +1,74 @@
 'use strict';
+
+const firstPromise = new Promise((resolve, reject) => {
+  let resolved = false;
+
+  const timeout = setTimeout(() => {
+    if (!resolved) {
+      reject(new Error('First promise was rejected'));
+    }
+  }, 3000);
```

Comment:
This timeout is used to reject the promise after 3 seconds if it's not resolved. But it's not necessary to use this timeout. Because if the promise is not resolved or rejected, it will be pending forever. It's not a problem. The Promise object will not consume any resources while it's pending.